Package.swift

```diff
@@ -449,7 +449,6 @@ extension Target.Dependency {
     static var AWSSDKIdentity: Self { "AWSSDKIdentity" }
     static var AWSSDKIdentityAPI: Self { "AWSSDKIdentityAPI" }
     static var AWSSDKChecksums: Self { "AWSSDKChecksums" }
```

Comment:
Above, the partitions file contained in the `AWSSDKPartitions` target is deleted.

In this file, just rendered changes (to remove `AWSSDKPartitions`) from the `Package.base` file above.